clang/lib/CodeGen/CGOpenMPRuntime.cpp

```diff
@@ -10343,6 +10343,23 @@ void CGOpenMPRuntime::emitTargetDataStandAloneCall(
         MapNamesArray,
         InputInfo.MappersArray.emitRawPointer(CGF)};
 
+    // Nowait calls have header declarations that take 13 arguments. Hence, the
```

Comment:
You can make `OffloadingArgs` above a `SmallVector<llvm::Value *>` and then push extra arguments if needed.